src/sql/workbench/browser/designer/designerActions.ts

```diff
@@ -0,0 +1,114 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Designer } from 'sql/workbench/browser/designer/designer';
+import { DesignerEdit, DesignerEditType, DesignerTableProperties } from 'sql/workbench/browser/designer/interfaces';
+import { Action } from 'vs/base/common/actions';
+import { localize } from 'vs/nls';
+
+export class AddColumnAction extends Action {
+	public static ID = 'tableDesigner.addColumnAction';
+	public static ICON = 'add-row-button new codicon';
+	public static LABEL = localize('tableDesigner.addColumnAction', 'Add new row');
```

Comment:
Please copy use the logic from previous button implementation.
1. the default text is Add New. 
3. the text of the add new button can be overwritten by the table configuration